go.mod

```diff
@@ -59,12 +57,13 @@ require (
 )
 
 replace (
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible
```

Comment:
what was the problem?

Reply:
I don't remember anymore, but some conflict in the versions. In any case, this was borrowed from Helm itself -> https://github.com/helm/helm/blob/master/go.mod#L49